message_sender/tests.py

```diff
@@ -427,15 +434,49 @@ def test_create_outbound_data_simple(self):
         self.assertIsNotNone(d.id)
         self.assertEqual(d.version, 1)
         self.assertEqual(str(d.to_addr), "+27123")
+        self.assertEqual(str(d.to_identity), "0c03d360")
         self.assertEqual(d.delivered, False)
         self.assertEqual(d.attempts, 1)
         self.assertEqual(d.metadata, {
             "voice_speech_url": "https://foo.com/file.mp3"
         })
         self.assertEqual(d.channel, None)
 
+    @responses.activate
+    def test_create_outbound_data_new_identity(self):
+
+        self.add_identity_search_response('+2712345', '0c03d360123', 0)
```

Comment:
Maybe make the second argument here `None`, so that it's more obvious that we're creating an empty set of results.